lib/docs/scrapers/rest_framework.rb

```diff
@@ -0,0 +1,26 @@
+module Docs
+  class RestFramework < Mkdocs
+    self.name = 'Django REST Framework'
+    self.release = '3.9.2'
+    self.slug = 'rest_framework'
+    self.type = 'mkdocs'
```

Comment:
Sorry, forgot to clarify when I commented earlier. This line can be removed since the `type` should be inherited from the `Mkdocs` class.
```suggestion
```

Reply:
Yeah, I forgot to remove it.